homeassistant/components/velux/__init__.py

```diff
@@ -23,9 +24,7 @@
 
 
 async def async_setup(hass, config):
-    """Set up the velux component."""
-    from pyvlx import PyVLXException
-
+    """Set up the velux component."""    
     try:
         hass.data[DATA_VELUX] = VeluxModule(hass, config)
```

Comment:
Please just pass `config[DOMAIN]` here. The `VeluxModule` class doesn't need to know about the whole config.